plato/draw/fresnel/ConvexPolyhedra.py

```diff
@@ -6,7 +6,7 @@
 from .FresnelPrimitive import FresnelPrimitive
 
 @ShapeDecorator
-class ConvexPolyhedra(draw.ConvexPolyhedra, FresnelPrimitive):
+class ConvexPolyhedra(FresnelPrimitive, draw.ConvexPolyhedra):
```

Comment:
Now that `FresnelPrimitive` is a mixin, we inherit from it _first_ (to put it early in the MRO) and then allow its call to `super().__init__` to forward all arguments as desired.